test/client.js

```diff
@@ -137,95 +139,6 @@ test('basic get with query params', (t) => {
   })
 })
 
-test('basic get with query params with object throws an error', (t) => {
```

Comment:
if this no longer throws an error, tests need to be adjusted to document new behaviour.
what was the handling of these edge cases based on?

Reply:
Well, this was unorthodox. `new URL`, `node:querystring` and `fast-querystring` supports both array and object as an input.

Reply:
can you add args of those type into existing happy-path test which used to pass args of all supported types?

Reply:
Thanks for the review. I truly appreciate it. I added a new test for this particular happy path.